### PR TITLE
Migrated keyword page to b3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun

### CODE REVIEW

File: corehq/apps/reminders/templates/reminders/keyword_list.html [hunk 1]
```diff
@@ -24,44 +24,47 @@
...
+                                {% endblocktrans %}
+                            </p>
+                        </div>
+                        <div class="modal-footer">
+                            <a href="#" class="btn btn-danger delete-item-confirm">
+                                <i class="fa fa-remove"></i> {% trans "Delete" %}
+                            </a>
+                            <a href="#" data-dismiss="modal" class="btn btn-default">{% trans "Cancel" %}</a>
+                        </div>
+                    </div>
```
Comment (gcapalbo) [inline corehq/apps/reminders/templates/reminders/keyword_list.html]: missing a `</div>` here

**Comment by gcapalbo** [comment]
I tested this and looks good overall, just a few things:
- the breadcrumbs are missing from the keyword list page (the Messaging > Keywords > etc.); it looks like they were also missing from the old page too - can you add them in?
- when i delete a keyword from the keyword list page, the modal disappears but the screen is still locked and i can't scroll or click on anything
- when creating / editing a structured keyword, the text for "Use Custom Delimiter", "Use Named Answers", and "Use Joining Character" are scrunched, can we give them more room?
- the text inputs for the custom delimiter and joining characters are a little too small (we should be able to read the entire placeholder text in them)
- if you leave the custom delimiter or the joining character text inputs blank, the form correctly throws a ValidationError but no error message is displayed

**Comment by biyeun** [comment]
@kkrampa more conflicts

**Comment by kkrampa** [comment]
@biyeun Updated

**Comment by biyeun** [comment]
@kkrampa conflicts, oy. Sorry was offline the last two days, will get to this immediately tomorrow after it's resolved

**Comment by gcapalbo** [comment]
@kkrampa looks good, just one thing - after i delete a keyword, the gray background now clears, but the scroll bars don't reappear (so if you have more than 10 keywords, you can't scroll to see them all)

**Comment by kkrampa** [comment]
@gcapalbo @biyeun Updated

**Comment by biyeun** [comment]
@kkrampa merge conflicts

**Comment by gcapalbo** [comment]
@biyeun this looks good from my side, was there anything else blocking or is it good to merge?

**Comment by biyeun** [comment]
looks good

**Pull request merged**